package.json

```diff
@@ -56,51 +63,51 @@
     "test/"
   ],
   "dependencies": {
-    "global": "^4.3.2",
-    "video.js": "^6 || ^7 || ^8"
+    "global": "^4.4.0"
   },
   "peerDependencies": {
     "video.js": "^6 || ^7 || ^8"
   },
   "devDependencies": {
+    "@babel/cli": "^7.13.16",
+    "@babel/runtime": "^7.14.0",
+    "@videojs/babel-config": "^0.2.0",
+    "@videojs/generator-helpers": "~2.0.2",
     "conventional-changelog-cli": "^2.0.1",
     "conventional-changelog-videojs": "^3.0.0",
     "doctoc": "^1.3.1",
     "husky": "^1.0.0-rc.13",
-    "jsdoc": "^3.6.10",
-    "karma": "^3.0.0",
+    "jsdoc": "^3.6.11",
+    "karma": "^6.3.2",
     "lint-staged": "^7.2.2",
     "not-prerelease": "^1.0.1",
     "npm-merge-driver-install": "^1.0.0",
     "npm-run-all": "^4.1.5",
     "pkg-ok": "^2.2.0",
-    "rollup": "^0.66.0",
+    "rollup": "^2.46.0",
     "semver": "^5.1.0",
     "shx": "^0.3.2",
-    "sinon": "^6.1.5",
-    "videojs-generate-karma-config": "^7.1.0",
-    "videojs-generate-rollup-config": "~2.2.0",
-    "videojs-generator-verify": "~1.0.3",
-    "videojs-standard": "~7.1.0"
+    "sinon": "^9.1.0",
+    "video.js": "^6 || ^7 || ^8",
+    "videojs-generate-karma-config": "~8.0.0",
+    "videojs-generate-rollup-config": "^7.0.0",
+    "videojs-generator-verify": "^4.1.0",
+    "videojs-standard": "^9.0.1"
   },
   "module": "dist/videojs-contrib-quality-levels.es.js",
```

Comment:
Does this value need to change, too? Or does this file still exist?

Reply:
This stays the same.